spec/watirspec/elements/button_spec.rb

```diff
@@ -75,6 +75,23 @@
       expect(browser.button(xpath: "//input[@type='button']")).to exist
     end
 
+    it "matches the specific type when locating by type" do
+      aggregate_failures do
```

Comment:
For how I work with Watir specs, it is just unnecessary. I fix things one at a time and rerun failing tests until they pass. Plus I'd rather see fewer blocks in our specs rather than adding a bunch more throughout all the files. :)